adb_graphics/specs.py

```diff
@@ -188,6 +188,26 @@ def lifted_index_colors(self) -> np.ndarray:
         ctable[15] = [1, 1, 1, 1]
         return ctable
 
+    @property
+    @lru_cache()
+    def mdn_colors(self) -> np.ndarray:
+
+        ''' Default color map for Max Downdraft '''
+
+        grays = cm.get_cmap('Greys', 2)([0])
+        others = cm.get_cmap('jet', 18)(range(18, 1, -1), alpha=0.6)
```

Comment:
The names of the colormaps should be set in default_specs.yml instead of hard coding here. This also applies to the other color definitions below.